Library/Formula/boot.rb

```diff
@@ -0,0 +1,18 @@
+require "formula"
+
+class Boot < Formula
+  homepage "http://boot-clj.com"
+  version "2.0.0-rc4"
+  url "https://github.com/boot-clj/boot/releases/download/2.0.0-rc4/boot.sh",
```

Comment:
The file we need is not in that archive and the build process is slightly complex.